pkgs/development/python-modules/hyppo/default.nix

```diff
@@ -16,7 +16,7 @@
 buildPythonPackage rec {
   pname = "hyppo";
   version = "0.4.0";
-  format = "setuptools";
+  pyproject = true;
```

Comment:
Thanks for applying this change.
Could you please add `build-system = [ setuptools ];` as well?

Reply:
Done!